<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mpvQC</string>
  </property>
  <widget class="QWidget" name="mainWindowContentWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="mainWindowContentSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mainWindowMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="mainWindowFileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewQcDocument"/>
    <addaction name="actionOpenQcDocuments"/>
    <addaction name="actionSaveQcDocument"/>
    <addaction name="actionSaveQcDocumentAs"/>
    <addaction name="separator"/>
    <addaction name="actionExitMpvQc"/>
   </widget>
   <widget class="QMenu" name="mainWindowVideoMenu">
    <property name="title">
     <string>Vi&amp;deo</string>
    </property>
    <addaction name="actionOpenVideoFile"/>
    <addaction name="actionOpenSubtitleFile"/>
    <addaction name="actionOpenNetworkStream"/>
    <addaction name="separator"/>
    <addaction name="actionResizeVideoToOriginalResolution"/>
   </widget>
   <widget class="QMenu" name="mainWindowSettingsMenu">
    <property name="title">
     <string>Optio&amp;ns</string>
    </property>
    <widget class="QMenu" name="menuWindowTitle">
     <property name="title">
      <string>Window title</string>
     </property>
     <addaction name="actionWindowTitleDefault"/>
     <addaction name="actionWindowTitleFile"/>
     <addaction name="actionWindowTitlePath"/>
    </widget>
    <widget class="QMenu" name="menuDocumentSettings">
     <property name="title">
      <string>Document settings</string>
     </property>
     <addaction name="actionSaveVideoPathToDocument"/>
     <addaction name="actionSaveNickNameToDocument"/>
    </widget>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="action_3"/>
    </widget>
    <addaction name="actionEditNickname"/>
    <addaction name="actionEditCommentTypes"/>
    <addaction name="separator"/>
    <addaction name="menuWindowTitle"/>
    <addaction name="actionDarkTheme"/>
    <addaction name="separator"/>
    <addaction name="menuLanguage"/>
    <addaction name="separator"/>
    <addaction name="actionEditMpvConf"/>
    <addaction name="actionEditInputConf"/>
    <addaction name="separator"/>
    <addaction name="menuDocumentSettings"/>
    <addaction name="actionDocumentBackups"/>
    <addaction name="actionDocumentBackupInterval"/>
   </widget>
   <widget class="QMenu" name="mainWindowAboutMenu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCheckForUpdates_2"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAboutMpvQc"/>
   </widget>
   <addaction name="mainWindowFileMenu"/>
   <addaction name="mainWindowVideoMenu"/>
   <addaction name="mainWindowSettingsMenu"/>
   <addaction name="mainWindowAboutMenu"/>
  </widget>
  <action name="actionNewQcDocument">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;New QC document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenQcDocuments">
   <property name="icon">
    <iconset theme="document-open-recent">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Open QC document(s) ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveQcDocument">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Save QC document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveQcDocumentAs">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>S&amp;ave QC document as ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExitMpvQc">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit mpvQC</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpenVideoFile">
   <property name="icon">
    <iconset theme="mpv">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open &amp;video ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionOpenNetworkStream">
   <property name="icon">
    <iconset theme="network-server">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open &amp;network stream ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Shift+O</string>
   </property>
  </action>
  <action name="actionResizeVideoToOriginalResolution">
   <property name="icon">
    <iconset theme="image-resize">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Resize video to original resolution</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCheckForUpdates">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="arrow-down">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Check For Updates ...</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionAboutMpvQc">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About &amp;mpvQC</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset theme="preferences-other">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionOpenSubtitleFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="media-view-subtitles-symbolic">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Open subtitle ...</string>
   </property>
  </action>
  <action name="actionEditCommentTypes">
   <property name="text">
    <string>Edit comment types ...</string>
   </property>
  </action>
  <action name="actionEditNickname">
   <property name="text">
    <string>Edit nickname ...</string>
   </property>
  </action>
  <action name="actionWindowTitleDefault">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Default title</string>
   </property>
  </action>
  <action name="actionWindowTitleFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Video file</string>
   </property>
  </action>
  <action name="actionWindowTitlePath">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Video path</string>
   </property>
  </action>
  <action name="actionDarkTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark theme</string>
   </property>
  </action>
  <action name="actionEditMpvConf">
   <property name="text">
    <string>Edit mpv.conf ...</string>
   </property>
  </action>
  <action name="actionEditInputConf">
   <property name="text">
    <string>Edit input.conf ...</string>
   </property>
  </action>
  <action name="actionDocumentBackups">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Document backups</string>
   </property>
  </action>
  <action name="actionDocumentBackupInterval">
   <property name="text">
    <string>Document backup interval ...</string>
   </property>
  </action>
  <action name="actionSaveNickNameToDocument">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save nickname</string>
   </property>
  </action>
  <action name="actionSaveVideoPathToDocument">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save video path</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string notr="true">__</string>
   </property>
  </action>
  <action name="actionCheckForUpdates_2">
   <property name="icon">
    <iconset theme="system-software-update">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Check for updates ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
